ans/ANS-104.md

```diff
@@ -221,6 +226,21 @@ A tag object is valid iff.:
 To write a bundle of DataItems, each DataItem should be constructed, signed, encoded, and placed in a transaction with
 the transaction body format and transaction tags specified in Section 1.
 
+#### 3.1 Nested bundle
+
+It is possible to construct data-items in such a way that they themselves are a bundle.
+The nested bundle and its nested data-items need to be verified and validated in the same manner
```

Comment:
I don't think the spec says anything about verifying a "bundle" per se. Rather it focuses on verifying its individual data items. I think this should instead focus on that aspect. Additionally, consider making the description of the verification process more instructive similar to the way I did above re: assembling a DataItem Bundle.

Reply:
I didn't go into the verification because it shouldn't be different in any way to verifying top level bundle or top level data-item. I refer to the other sections of the document in how bundle and data-items are verified. The word "verification" on bundle is also misleading, since it only needs to follow specification format, which I refer to with a link. Verifying with confirming cryptographic signature only apply to data-items.